### PR TITLE
Add Carousel to the webapps section

### LINKED ISSUE #136: Show and highlight web apps on OpenMS website
We generate our website with gohugo. Hugo will in general convert markdown .md and restructured text .rst files into html. For custom styling we still need html code sections, which are in our partials folder: https://github.com/OpenMS/OpenMS-website/tree/main/layouts/partials

We would need some nice way to present our web apps on the website, with mainly a name and some kind of logo. We can also remove the overview of proteomics, metabolomics and workflows, which gets its structrur from hero items in the partials folder. I would recommend looking at examples from different website that also use gohugo how to create a html partial https://gohugo.io/showcase/. Because we want to add more apps in time, we probably need some scroll or search function for the apps.

We can also make use of the hero item partial and change it.

### PR DESCRIPTION
**Pull request by hjn0415a**
### User description

**Brief description of what is fixed or changed**
fixes #136 

Describe your changes here
--> Adjust the size of swiper-slide for mobile screens.

***

### PR Type
enhancement, documentation

***

### Description

* Integrated a new carousel component using Swiper.js to highlight web applications on the OpenMS website.
* Added Bootstrap and jQuery libraries to enhance UI and functionality.
* Created a new section for web apps with dynamic loading from configuration.
* Updated styles and scripts to support the new carousel and web apps display.
* Simplified the key features section layout by removing feature box content.

___

### **PR Type**
Enhancement, Documentation

___

### **Description**
- Integrated a new carousel component using Swiper.js to highlight web applications on the OpenMS website.
- Added jQuery, Bootstrap, and Swiper.js dependencies to enhance UI and functionality.
- Created a new section for web apps with dynamic loading from configuration.
- Updated styles and scripts to support the new carousel and web apps display.
- Simplified the key features section layout by removing feature box content.

___

### **Changes walkthrough** 📝
<table><thead><tr><th></th><th align="left">Relevant files</th></tr></thead><tbody><tr><td><strong>Dependencies</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>baseof.html</strong><dd><code>Add jQuery, Bootstrap, and Swiper.js scripts</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

layouts/_default/baseof.html

- Added jQuery, Bootstrap, and Swiper.js scripts.

</details>

  </td>
  <td><a href="https://github.com/OpenMS/OpenMS-website/pull/165/files#diff-ec6e7cb6e89cbe10d49085811bd67ad23df375247295fc387da90f5fdc56de7f">+3/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>css.html</strong><dd><code>Add Swiper CSS link</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

layouts/partials/css.html

- Added Swiper CSS link.

</details>

  </td>
  <td><a href="https://github.com/OpenMS/OpenMS-website/pull/165/files#diff-e667f836a02d67217a1cf419d21974e3ff9945d82235e2310033f9c3e6dd45f6">+1/-0</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Enhancement</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>index.html</strong><dd><code>Integrate web apps section and enable case studies</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

layouts/index.html

<li>Integrated new <code>webapps.html</code> partial.<br> <li> Enabled <code>casestudies</code> section.<br>

</details>

  </td>
  <td><a href="https://github.com/OpenMS/OpenMS-website/pull/165/files#diff-96762550561f76f989ced99c4a0751e94696682490ee94b2d77a66377c9619eb">+4/-2</a>&nbsp; &nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>keyfeatures.html</strong><dd><code>Simplify key features section layout</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

layouts/partials/keyfeatures.html

- Removed feature box content.

</details>

  </td>
  <td><a href="https://github.com/OpenMS/OpenMS-website/pull/165/files#diff-6d86986970fb100aa7691dfe90c64eede9c787efbf649134cc9c03dcb997fa02">+1/-11</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>webapps.html</strong><dd><code>Create web apps section with Swiper</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

layouts/partials/webapps.html

- Created new web apps section using Swiper.

</details>

  </td>
  <td><a href="https://github.com/OpenMS/OpenMS-website/pull/165/files#diff-d41c7f7aa4e3f2ed313d36350e70f2dd54f7d1750c562b54aaf0222871ae12a7">+22/-0</a>&nbsp; &nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>webapps.css</strong><dd><code>Add styles for web apps carousel</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

assets/css/webapps.css

<li>Added styles for web apps carousel.<br> <li> Included responsive design adjustments.<br>

</details>

  </td>
  <td><a href="https://github.com/OpenMS/OpenMS-website/pull/165/files#diff-e939772f2afbebdb74e2c15282868d989a18c1c1668bf0c1014b10c3206590b5">+125/-0</a>&nbsp; </td>

</tr>                    

<tr>
  <td>
    <details>
      <summary><strong>swiper.js</strong><dd><code>Initialize Swiper and add slide interactions</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

assets/js/swiper.js

<li>Initialized Swiper with navigation and breakpoints.<br> <li> Added event listeners for slide interactions.<br>

</details>

  </td>
  <td><a href="https://github.com/OpenMS/OpenMS-website/pull/165/files#diff-001e68775dfec92fda4d7c021564d6b51d67da350055e23cabb4f11145dc24be">+78/-0</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr><tr><td><strong>Configuration changes</strong></td><td><table>
<tr>
  <td>
    <details>
      <summary><strong>config.yaml</strong><dd><code>Add web apps configuration for carousel</code>&nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; &nbsp; </dd></summary>
<hr>

config.yaml

- Added web apps configuration for carousel.

</details>

  </td>
  <td><a href="https://github.com/OpenMS/OpenMS-website/pull/165/files#diff-d8d0422389f03d783e32e627250fe29834bd09c6361640d1ff00661dd6820034">+34/-0</a>&nbsp; &nbsp; </td>

</tr>                    
</table></td></tr></tr></tbody></table>

___

> 💡 **PR-Agent usage**: Comment `/help "your question"` on any pull request to receive relevant information

### CODE REVIEW

**Comment by JKVISION1101** [comment]
@matteopilz @jpfeuffer 

Our team created new pull request about webapps on OpenMSwebiste using carousel style.

File: assets/js/swiper.js
Comment (jpfeuffer) [inline assets/js/swiper.js:1]: Can you add some small comment to the beginning of the file to describe what this is for?

"Configures swipeable cards for features like webapps.  Requires swiper.bundle.js to be loaded from a CDN first. See layouts/_defaults/base_of.html"

Comment (hjn0415a) [reply assets/js/swiper.js:1]: do you mean to leave a comment in the assets/js/swiper.js file? and,
Is the following sentence an example of a comment?

"Configures swipeable cards for features like webapps. Requires swiper.bundle.js to be loaded from a CDN first. See layouts/_defaults/base_of.html"

Comment (jpfeuffer) [reply assets/js/swiper.js:1]: Both, yes :)

Comment (hjn0415a) [reply assets/js/swiper.js:1]: Got it after adding a comment, I'll send the PR again.

**Pull request closed without merging**